io.catenax.item_stock/2.0.0/ItemStock.ttl

```diff
@@ -0,0 +1,174 @@
+    samm:characteristic :PositionsCollection.
+:materialNumberCustomer a samm:Property;
+    samm:preferredName "Customer Material Number"@en;
+    samm:description "Material identifier as assigned by customer. This material number identifies the material (as planned) in customer's database. For standard parts (e.g. screws) for which a customer does not have an own number, the materialNumberSupplier should be set in this field, too."@en;
+    samm:characteristic :MaterialNumberCharacteristic;
+    samm:exampleValue "MNR-7307-AU340474.002".
+:materialNumberSupplier a samm:Property;
+    samm:preferredName "Supplier Material Number"@en;
+    samm:description "Material identifier as assigned by supplier. This material number identifies the material (as planned) in supplier's database."@en;
+    samm:characteristic :MaterialNumberCharacteristic;
+    samm:exampleValue "MNR-8101-ID146955.001".
+:PositionsCollection a samm-c:Collection;
+    samm:preferredName "Positions Collection"@en;
+    samm:description "If any stock is available, then there is at least one position. \n\nIn case of a supplier's stock for a customer, the supplier can  differentiate the stock according to the order positions of his customer. In this case multiple positions can be given. There could also be only one position without an OrderPositionReference indicating an anonymous stock which is allocated to the customer.\n\nIn case of a customer's stock delivered by a supplier, there is exactly one position without order position references."@en;
+    samm:dataType :Position.
+:MaterialNumberCharacteristic a samm:Characteristic;
+    samm:preferredName "Material Number"@en;
+    samm:description "The material number is a multi-character string, usually assigned by an ERP system."@en;
+    samm:dataType xsd:string.
+:Position a samm:Entity;
+    samm:preferredName "Position"@en;
+    samm:description "The Position can be located at several stocks. \nIn case of a supplier's stock for a customer, a position may be either anonymous or reference a position within a customer order. \nIn case of a customer's stock for a supplier, the order position reference MUST NOT be set."@en;
+    samm:properties ([
+  samm:property :orderPositionReference;
+  samm:optional "true"^^xsd:boolean
+] :lastUpdatedOnDateTime :allocatedStocks).
+:orderPositionReference a samm:Property;
+    samm:preferredName "Order Position Reference"@en;
+    samm:description "The order position reference contains information to identify a position within an order."@en;
+    samm:characteristic :OrderPositionReferenceCharacteristic.
+:lastUpdatedOnDateTime a samm:Property;
+    samm:preferredName "Lasted Updated on Date"@en;
+    samm:description "Date and time at which the position has been updated by the supplier's systems. Date and time are set according to a timezone."@en;
+    samm:characteristic :DateTimeStamp;
+    samm:exampleValue "2023-04-01T14:23:00"^^xsd:date.
```

Comment:
The Characteristic used is xsd:date. Either you need to change the example value to only date or change the Characteristic.